tasks/1_golf-score/golf-score.js

```diff
@@ -0,0 +1,33 @@
+const GOLF_SCORES = {
+  HOLE_IN_ONE: 'Hole-in-one!',
+  EAGLE: 'Eagle',
+  BIRDIE: 'Birdie',
+  PAR: 'Par',
+  BOGEY: 'Bogey',
+  DOUBLE_BOGEY: 'Double Bogey',
+  GO_HOME: 'Go Home!',
+};
+
+function golfScore(par, strokes) {
```

Comment:
Please, use more friendly naming for a functions and params.
When you coding - you have to make naming of classes, variables, functions,... so clear that your teammates will understand what it is at the first sight.
In this case i don't understand what `golfScore` is (the same with params).